src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/MLDsa/MLDsaTestsData.cs

```diff
@@ -57,15 +62,15 @@ public override string ToString() =>
     public class MLDsaNistTestCase
     {
         public int NistTestCaseId { get; init; }
-        public MLDsaAlgorithm Algorithm { get; init; }
-        public bool ShouldPass { get; init; }
+        public required MLDsaAlgorithm Algorithm { get; init; }
+        public required bool ShouldPass { get; init; }
 
```

Comment:
Why?  We don't use `init` or `required` anywhere else in crypto.

Reply:
I think `init` will be needed by the compiler anyway for the primary constructor on the record. I'd say that's worth it to keep `IsExternalInit.cs` to support that so we can have a ctor with just the required properties and let the rest of the boilerplate be handled by the compiler.